core/src/main/scala/kafka/admin/ConsumerGroupCommand.scala

```diff
@@ -291,14 +291,21 @@ object ConsumerGroupCommand extends Logging {
     }
 
     private def printStates(states: Map[String, GroupState]): Unit = {
-      for ((groupId, state) <- states) {
-        if (shouldPrintMemberState(groupId, Some(state.state), Some(1))) {
+      val stateProps =  states.filter{case(groupId,state)=>shouldPrintMemberState(groupId, Some(state.state), Some(1))}
+        .map{case (_,state)=>
           val coordinator = s"${state.coordinator.host}:${state.coordinator.port} (${state.coordinator.idString})"
```

Comment:
I suggest to extract this as a method `coordinatorString` in the `GroupState` case class or in an inner function.

Reply:
Thanks for the suggestion, I've extracted this into the `GroupState` case class.